pkg/noderesourcetopology/plugin.go

```diff
@@ -0,0 +1,74 @@
+package noderesourcetopology
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/scheduler/framework"
+	apiconfig "sigs.k8s.io/scheduler-plugins/pkg/apis/config"
+
+	topologyv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+)
+
+// TopologyMatch plugin which run simplified version of TopologyManager's admit handler
+type TopologyMatch struct {
+	nodeResTopologyPlugin
```

Comment:
I'd prefer to unroll `nodeResTopologyPlugin` here, and remove noderesourcetopology/types.go.